_articles/custom-mouse-buttons.md

```diff
@@ -1,20 +1,51 @@
 ---
 layout: article
-title: Setting Custom Mouse Actions
+title: Customizing Your Mouse
 description: >
-   How to set custom mouse actions with xbindkeys
+   How to custom the buttons on your mouse
```

Comment:
`custom` is not a verb, this should probably say `How to customize`.

Reply:
Should be fixed now.